src/component/mxgraph/GraphConfigurator.ts

```diff
@@ -63,33 +61,10 @@ export default class GraphConfigurator {
   }
 
   private configureNavigationSupport(options: GlobalOptions): void {
-    const panningHandler = this.graph.panningHandler;
     if (options?.navigation?.enabled) {
-      // Pan configuration
-      panningHandler.addListener(mxgraph.mxEvent.PAN_START, this.getPanningHandler('grab'));
-      panningHandler.addListener(mxgraph.mxEvent.PAN_END, this.getPanningHandler('default'));
-
-      panningHandler.usePopupTrigger = false; // only use the left button to trigger panning
-      // Reimplement the function as we also want to trigger 'panning on cells' (ignoreCell to true) and only on left-click
-      // The mxGraph standard implementation doesn't ignore right click in this case, so do it by ourselves
-      panningHandler.isForcePanningEvent = (me): boolean => mxgraph.mxEvent.isLeftMouseButton(me.getEvent()) || mxgraph.mxEvent.isMultiTouchEvent(me.getEvent());
-      this.graph.setPanning(true);
-
-      // Zoom configuration
-      this.graph.registerMouseWheelZoomListeners(options.navigation.zoom);
+      this.graph.enableNavigation(options.navigation.zoom ?? {});
```

Comment:
**suggestion:** 

This change is unnecessary because the `options.navigation.zoom ?? {}` expression is already handled by the `ensureValidZoomConfiguration()` function. 

Therefore, it can be simplified to just `this.graph.enableNavigation(options.navigation.zoom);`. 
Keeping the code concise and avoiding redundant checks leads to cleaner and more maintainable code.